QuizApp/QuizApp/Login/EmailView.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+import SnapKit
+
+class EmailView: UIView {
+
+    var delegate: EmailViewDelegate!
```

Comment:
This will cause a memory leak because of circular reference. Please read the documentation on the delegate pattern.